renderdoc/core/sparse_page_table.cpp

```diff
@@ -913,7 +913,7 @@ template <typename SerialiserType>
 void DoSerialise(SerialiserType &ser, Sparse::Page &el)
```

Comment:
Was not sure about marking up helper structures but it seemed to fit the criteria of an offset or size measured in bytes

Reply:
It's probably better to err on the side of caution and annotate these just in case, but really we should be hiding these structures & chunks entirely from the user. Anything that doesn't come from the API itself is liable to cause confusion.